jmh-core-ct/src/test/java/org/openjdk/jmh/ct/InMemoryGeneratorDestination.java

```diff
@@ -60,6 +60,11 @@ public InputStream getResource(String resourcePath) throws IOException {
 
     @Override
     public Writer newClass(String className) throws IOException {
+        return newClass(className, null);
+    }
```

Comment:
Here and later. I don't think you need this override. Just pass `null` where needed. This would save us from the accidental passing of `null` where originating element is actually available.

Reply:
Fair enough. Sorry about that, I'm just used to being defensive about making backwards incompatible changes. 😄